clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java

```diff
@@ -422,4 +425,90 @@ public void sendHeartBeats(HeartbeatsRequest heartbeatsRequest) {
           "heartbeats fails ", e);
     }
   }
+
+  /**
+   * Get schemas list of service
+   *
+   * @param serviceId
+   * @return
+   * @throws OperationException
+   */
+  public List<GetSchemaResponse> getServiceSchemasList(String serviceId) {
+    try {
+      HttpResponse response = httpClient
+          .getHttpRequest("/registry/microservices/" + serviceId + "/schemas", null, null);
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode jsonNode = mapper.readTree(response.getContent());
+        return mapper.readValue(jsonNode.get("schemas").toString(), new TypeReference<List<GetSchemaResponse>>() {
+        });
+      } else {
+        throw new OperationException(
+            "get service schemas list fails, statusCode = " + response.getStatusCode() + "; message = " + response
+                .getMessage()
+                + "; content = " + response.getContent());
+      }
+    } catch (IOException e) {
+      throw new OperationException(
+          "get service schemas list fails", e);
+    }
+  }
+
+  /**
+   * Get one schema context of service
+   *
+   * @param serviceId
+   * @param schemaId
+   * @return
+   * @throws OperationException
+   */
+  public String getServiceSchemaContext(String serviceId, String schemaId) {
+    try {
+      HttpResponse response = httpClient
+          .getHttpRequest("/registry/microservices/" + serviceId + "/schemas/" + schemaId, null, null);
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode jsonNode = mapper.readTree(response.getContent());
+        return jsonNode.get("schema").textValue();
+      } else {
+        throw new OperationException(
+            "get service schema context fails, statusCode = " + response.getStatusCode() + "; message = " + response
+                .getMessage()
+                + "; content = " + response.getContent());
+      }
+    } catch (IOException e) {
+      throw new OperationException(
+          "get service schemas context fails", e);
+    }
+  }
+
+  /**
+   * update schema context of service
+   *
+   * @param serviceId
+   * @param schemaId
+   * @param schemaResponse
+   * @return
+   * @throws OperationException
+   */
+  public Boolean updateServiceSchemaContext(String serviceId, String schemaId, GetSchemaResponse schemaResponse) {

```

Comment:
Maybe ```boolean``` is better.

Reply:
Thanks for your suggestion, it has been modified.